classes/Customer.php

```diff
@@ -186,7 +186,7 @@ class CustomerCore extends ObjectModel
             'optin' => array('type' => self::TYPE_BOOL, 'validate' => 'isBool'),
             'website' => array('type' => self::TYPE_STRING, 'validate' => 'isUrl'),
             'company' => array('type' => self::TYPE_STRING, 'validate' => 'isGenericName'),
-            'siret' => array('type' => self::TYPE_STRING, 'validate' => 'isSiret'),
+            'siret' => array('type' => self::TYPE_STRING, 'validate' => 'isGenericName'),
```

Comment:
Hello @fatmaBouchekoua 
Why did you not adapt isSiret function ? https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Validate.php#L1092

WDYT @maximebiloe